aptos-move/aptos-vm-types/src/tests/test_change_set.rs

```diff
@@ -1,690 +1,678 @@
-/// |      3       |                |    create 103  |    create 103   |
-/// |      4       |                |    modify 104  |    modify 104   |
-/// |      5       |                |    delete      |    delete       |
-/// |      6       |    create 6    |    create 106  |    ERROR        |
-/// |      7       |    create 7    |    modify 107  |    create 107   |
-/// |      8       |    create 8    |    delete      |                 |
-/// |      9       |    modify 9    |    create 109  |    ERROR        |
-/// |      10      |    modify 10   |    modify 110  |    modify 110   |
-/// |      11      |    modify 11   |    delete      |    delete       |
-/// |      12      |    delete      |    create 112  |    modify 112   |
-/// |      13      |    delete      |    modify 113  |    ERROR        |
-/// |      14      |    delete      |    delete      |    ERROR        |
-/// *--------------*----------------*----------------*-----------------*
-/// |      15      |    +15         |                |    +15          |
-/// |      16      |                |    +116        |    +116         |
-/// |      17      |    +17         |    +117        |    +134         |
-/// *--------------*----------------*----------------*-----------------*
-/// |      18      |    create 18   |    +118        |    create 136   |
-/// |      19      |    modify 19   |    +119        |    modify 138   |
-/// |      20      |    delete      |    +120        |    ERROR        |
-/// *--------------*----------------*----------------*-----------------*
-/// |      21      |    +21         |    create 121  |    ERROR        |
-/// |      22      |    +22         |    modify 122  |    modify 122   |
-/// |      23      |    +23         |    delete      |    delete       |
-/// *--------------*----------------*----------------*-----------------*
-/// ```
-
-macro_rules! resource_write_set_1 {
-    ($d:ident) => {
-        vec![
-            mock_create_with_layout(format!("0{}", $d), 0, None),
-            mock_modify_with_layout(format!("1{}", $d), 1, None),
-            mock_delete_with_layout(format!("2{}", $d)),
-            mock_create_with_layout(format!("7{}", $d), 7, None),
-            mock_create_with_layout(format!("8{}", $d), 8, None),
-            mock_modify_with_layout(format!("10{}", $d), 10, None),
-            mock_modify_with_layout(format!("11{}", $d), 11, None),
-            mock_delete_with_layout(format!("12{}", $d)),
-        ]
-    };
-}
-
-macro_rules! module_write_set_1 {
-    ($d:ident) => {
-        vec![
-            mock_create(format!("0{}", $d), 0),
-            mock_modify(format!("1{}", $d), 1),
-            mock_delete(format!("2{}", $d)),
-            mock_create(format!("7{}", $d), 7),
-            mock_create(format!("8{}", $d), 8),
-            mock_modify(format!("10{}", $d), 10),
-            mock_modify(format!("11{}", $d), 11),
-            mock_delete(format!("12{}", $d)),
-        ]
+#[cfg(test)]
```

Comment:
argh github annotates your change here terribly

I trust you know what you changed, if you want review - comment on the part of the code that actually got changed